db/tasks.php

```diff
@@ -27,26 +27,13 @@
 
 /* List of handlers */
 global $CFG;
-$hearbeatmailminute = 'R';
-if ($CFG->branch < 28) {
-    $hearbeatmailminute = mt_rand(0, 59);
-}
 
 $tasks = array(
     array(
         'classname' => 'block_disk_quota\task\get_disk_usage',
         'blocking' => 0,
-        'minute' => '*/10',
-        'hour' => '*',
-        'day' => '*',
-        'dayofweek' => '*',
-        'month' => '*'
-    ),
-    array(
-        'classname' => 'block_disk_quota\task\send_heartbeat_email',
-        'blocking' => 0,
-        'minute' => $hearbeatmailminute,
-        'hour' => '*/12',
+        'minute' => '0',
```

Comment:
maybe it would make sense to put the minute as random here to avoid having all instances putting load on the db at the same time: https://moodledev.io/docs/4.4/apis/commonfiles/db-tasks.php#date-and-time-fields